Makefile

```diff
@@ -30,8 +33,11 @@ robot.jar:
 robot: robot.jar
 	curl -L -O https://raw.githubusercontent.com/ontodev/robot/master/bin/robot && chmod +x robot
 
-ontologies-merged.ttl: robot ontologies.ofn
-	ROBOT_JAVA_ARGS=-Xmx$(MEMORY) ./robot merge -i ontologies.ofn -o ontologies-merged.ttl
+ontologies-merged-original.ttl: robot ontologies.ofn
+	ROBOT_JAVA_ARGS=-Xmx$(MEMORY) ./robot merge -i ontologies.ofn -o ontologies-merged-original.ttl
+
+ontologies-merged.ttl: ontologies-merged-original.ttl manually_added.ttl
+	cat ontologies-merged-original.ttl manually_added.ttl > ontologies-merged.ttl
```

Comment:
This is not really a safe way to concatenate RDF files. Any blank nodes with the same identifier in the two files will be unified into one node in the new file.

Reply:
Fixed in 7df7058. I didn't realize that robot could do this since manually_added.ttl isn't an OWL ontology, but it seems to work. Robot appears to ignore the [owl:Axioms at the end of the file](https://github.com/NCATS-Gamma/omnicorp/blob/7df7058fb47ae9db758f46ad625545b15cf8b562/manually_added.ttl#L31-L106), but they're not really needed, and since this is just a temporary solution until the NCBI Taxonomy ontology is updated, I think it's fine for now.

Reply:
You could make it an (anonymous) OWL ontology by adding a triple to the file: `_:ont a owl:Ontology .` 😉

Reply:
I'm not sure how that will affect the parsing issue you're seeing.

Reply:
It seems to work as is, so I guess that's not needed. This might be because it's [a subclass of a class that *is* in the ontology](https://github.com/NCATS-Gamma/omnicorp/blob/7df7058fb47ae9db758f46ad625545b15cf8b562/manually_added.ttl#L14-L15), or Robot might just be merged everything at the RDF level.